views/random-beer.hbs

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
You only need the `<html>`, ` <head>` and `<body>` in the layout.hbs
The rest of the hbs files will be just the divs you need.